core/src/main/python/wlsdeploy/tool/deploy/deployer.py

```diff
@@ -409,6 +416,22 @@ def __process_archive_entry(self, location, key, value):
         self.logger.exiting(class_name=self._class_name, method_name=_method_name, result=result)
         return result
 
+    def add_application_attributes(self, model, location):
```

Comment:
I think this and deploy_app_attributes should have 'online' in the method names.
Also a comments to clarify that we're updating attributes that are not handled during online app deployment.